net/aircrack-ng/Makefile

```diff
@@ -36,6 +38,19 @@ define Package/aircrack-ng/description
   WLAN tools for breaking 802.11 WEP/WPA keys
 endef
 
+define Package/airmon-ng
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=@(PACKAGE_wireless-tools||PACKAGE_iw) +ethtool +procps +CONFIG_USB_SUPPORT:usbutils +CONFIG_PCI_SUPPORT:pciutils
```

Comment:
Now I am not sure why I add iw in the first place, it's not there before the split.

Nontheless, I tend to think that including wireless-tools is fine, because airmon-ng allows it even though it is almost deprecated by iw